pkg/inst/tests/test_sparkSQL.R

```diff
@@ -0,0 +1,36 @@
+library(testthat)
+
+context("SparkSQL functions")
+
+# Tests for jsonFile, registerTempTable, sql, count, table
+
+sc <- sparkR.init()
+
+sqlCtx <- sparkRSQL.init(sc)
+
+mockLines <- c("{\"Name\":\"Michael\"}",
+               "{\"Name\":\"Andy\", \"Age\":30}",
+               "{\"Name\":\"Justin\", \"Age\":19}")
+jsonPath <- tempfile(pattern="sparkr-test", fileext=".tmp")
+writeLines(mockLines, jsonPath)
+
+test_that("jsonFile() on a local file returns a DataFrame", {
+  df <- jsonFile(sqlCtx, jsonPath)
+  expect_true(inherits(df, "DataFrame"))
+  expect_true(count(df) == 3)
+})
+
+test_that("registerTempTable() results in a queryable table and sql() results in a new DataFrame", {
+  registerTempTable(df, "table1")
```

Comment:
So there is one last problem here - The variable `df` created in the previous test does not get carried over to the next test. Could you add a line that again creates `df` in this test case ?